scripts/update/radarr.sh

```diff
@@ -44,8 +44,17 @@ if [[ -f /install/.radarr.lock ]]; then
             fi
 
         elif [[ $(_radarr_version) = "mono-v2" ]]; then
-            echo_warn "Please upgrade your radarr to v3. It will be migrated to .Net core on the next \`box update\` run"
+            echo_warn "Radarr v0.2 is EOL and not supported. Please upgrade your radarr to v3. It will be migrated to .Net core on the next \`box update\` run"
             echo_docs "applications/radarr#migrating-to-v3-on-net-core"
         fi
     fi
+    #Mandatory SSL Port change for Readarr
+    sslport=$(grep -oPm1 "(?<=<SslPort>)[^<]+" /home/"$radarrOwner"/.config/Radarr/config.xml)
+    sslenabled=$(grep -oPm1 "(?<=<EnableSsl>)[^<]+" /home/"$radarrOwner"/.config/Radarr/config.xml)
+    if $sslport '==' '8787'; then
```

Comment:
not sure what that syntax was or if that would even work...
```suggestion
    if [ $sslport = '8787' ]; then
```